### PR TITLE
Updated documentation

### PR DESCRIPTION
**Pull request by naddah-5**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,3 +73,25 @@ flask --app src/app.py --debug run
...
+* Takes
+  * "image_list": [Base64, ...]
+    * Currently only uses the first image from the list
+  * "shape": [int, int, int]
+    * shape order is, [height, width, channels]
+  * "dtype": \<str>
+    * a string representation of a numpy [dtyp](https://numpy.org/doc/stable/reference/arrays.dtypes.html)
+* Returns
+  * "prediction": \<str>
+    * string representation of predictade gesture, or fail state
```
Comment (Rickos99) [inline README.md]: Did you mean predicted...

**Review by Rickos99 — CHANGES_REQUESTED** [review]
Typo, see above comments

File: README.md (hunk 1 above)
Comment (naddah-5) [reply README.md]: ooof

**Review by Rickos99 — APPROVED** [review]
Looks good

**Pull request merged**